src/main/resources/swagger/api-docs.yaml

```diff
@@ -5004,20 +5018,6 @@ paths:
 ## PARAMETERS
 ##########################################################################################
 parameters:
-  workflowTypeParam:
-    name: workflowType
-    type: string
-    description: The workflow language for the file you submitted. Cromwell currently supports WDL and CWL.
-    enum: [WDL, CWL]
-    required: false
-    in: formData
-  workflowTypeVersionParam:
-    name: workflowTypeVersion
-    description: The specification version for the workflow language being used. For WDL, Cromwell currently supports draft-2 and 1.0. For CWL, Cromwell currently supports v1.0.
-    required: false
-    type: string
-    enum: ["draft-2", "1.0", "v1.0"]
-    in: formData
```

Comment:
swagger can't handle having the `in: formData` defined here in `parameters`, because it can't validate the `consumes` settings for these. So I moved them up to where they're actually used.